src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -219,15 +221,38 @@ export default function TokenDetails({
       <TokenDetailsLayout>
         {detailedToken && !isPending ? (
           <LeftPanel>
-            <BreadcrumbNavLink to={`${isInfoExplorePageEnabled ? '/explore' : ''}/tokens/${chain.toLowerCase()}`}>
-              <ArrowLeft data-testid="token-details-return-button" size={14} /> Tokens
-            </BreadcrumbNavLink>
+            {isInfoTDPEnabled ? (
+              <BreadcrumbNav isInfoTDPEnabled>
+                <BreadcrumbNavLink to={`${isInfoExplorePageEnabled ? '/explore' : ''}/tokens/${chain.toLowerCase()}`}>
+                  <Trans>Explore</Trans> <ChevronRight data-testid="token-details-return-button" size={14} />{' '}
+                  <Trans>Tokens</Trans> <ChevronRight data-testid="token-details-return-button" size={14} />
```

Comment:
should these two be individually clickable? `Explore` to `/explore` and `Tokens` to `/explore/tokens/chain`?

Reply:
hm, let me double check with design on this. But in my mind, it should be the same link since `/explore` defaults to the tokens tab anyway -- and a user would expect it to open the chain-specific page.

Reply:
will double check for PDP nav